srcpkgs/python3-matplotlib/INSTALL.msg

```diff
@@ -1,3 +1,12 @@
+NOTICE: /etc/matplotlibrc is no longer supported as configuration file
```

Comment:
25-line won't even fit into nvidia's 80x25 TTY. Could it be shorter?

Reply:
What about leaving just the first line?
`NOTICE: /etc/matplotlibrc is no longer supported as configuration file`

Reply:
I could put it below, in the part about the backend

Reply:
ok, the rest can be `vdoc`'ed

Reply:
what was the convention for the file to `vdoc`? `README.voidlinux`?

Reply:
yes, `README.voidlinux`

Reply:
done